tensorflow/java/src/main/java/org/tensorflow/AbstractOperation.java

```diff
@@ -60,18 +60,28 @@ public String toString() {
   abstract long getUnsafeNativeHandle(int outputIdx);
 
   /**
-   * Returns the shape of the tensor of the {code outputIdx}th output of this operation.
+   * Returns the shape of the tensor of the {@code outputIdx}th output of this operation.
    *
    * @param outputIdx index of the output of this operation
    * @return output tensor shape
    */
   abstract long[] shape(int outputIdx);
 
   /**
-   * Returns the datatype of the tensor of the {code outputIdx}th output of this operation.
+   * Returns the datatype of the tensor of the {@code outputIdx}th output of this operation.
    *
    * @param outputIdx index of the output of this operation
    * @return output tensor datatype
    */
   abstract DataType dtype(int outputIdx);
+
+  /**
+   * Returns the tensor of the {@code outputIdx}th output of this operation.
+   * 
+   * <p>This is only supported in an eager execution environment.
+   * 
+   * @param outputIdx index of the output of this operation
+   * @return output tensor
```

Comment:
what does it return if the user happens to call it in a non-eager env?

Reply:
See `GraphOperation` class in this PR, it throws an `IllegalStateException` error